### ISSUE TITLE
Dimension error

### ISSUE THREAD

**Issue by davidlamb** [dhermes/bezier#119]
I was experimenting with the bezier library to create a curve in matplotlib. I installed using pip into a 64-bit anaconda python 2.7 installation (numpy version 1.14.2 and bezier version 0.8.0). Using a Jupyter Notebook and the code I copied from the documentation:

```python
import bezier
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline
print np.__version__
print bezier.__version__

nodes = np.asfortranarray([
    [0.0, 0.0],
    [1.5, 1.5],
    [3.0, 0.0],
])
curve = bezier.Curve(nodes, degree=2)
ax = curve.plot(100)
```

With this code I keep getting the error 

```
NotImplementedError: ('2D is the only supported dimension', 'Current dimension', 3L)
```

at the point of curve.plot(100). It appears that when the curve is created, it assigns a dimension of 3 instead of the dimension of 2. I tried to manually set the dimension (curve._dimension) but it just plots an empty axes.

Any thoughts?

**Comment by dhermes** [comment]
@davidlamb It's great to hear from users, thanks for stopping by!

In the [`0.7.0` release][1] I transposed the data structure (since accessing the points as columns should be contiguous in memory). Sorry for this inconvenience.

So you want:

```python
In [1]: import numpy as np

In [2]: import bezier

In [3]: nodes = np.asfortranarray([
   ...:     [0.0, 1.5, 3.0],
   ...:     [0.0, 1.5, 3.0],
   ...: ])

In [4]: curve = bezier.Curve(nodes, degree=2)

In [5]: ax = curve.plot(100)

In [6]: ax.figure.show()
```

which will produce something like

![figure_1](https://user-images.githubusercontent.com/520669/37789414-0dd9dbe6-2dc1-11e8-8802-d16ddbd0e2af.png)

----

If you have lots of code already, a "simple" fix is just to make a C-ordered array, and then take the transpose (which well then be Fortran-ordered), e.g.

```python
nodes = np.array([
    [0.0, 0.0],
    [1.5, 1.5],
    [3.0, 0.0],
], order='C').T
```

----

I am going to preemptively close this issue since it seems I have resolved your question.

[1]: http://bezier.readthedocs.io/en/latest/releases/0.7.0.html#data-structures
